src/OversetSimulation.h

```diff
@@ -109,6 +111,30 @@ class OversetSimulation
         m_is_adaptive_holemap_alg = alg;
         if (m_is_adaptive_holemap_alg == true) m_tg.setHoleMapAlgorithm(1);
     }
+
+    void set_composite_num(int num_composite)
+    {
+        m_num_composite_bodies = num_composite;
+        m_tg.setNumCompositeBodies(num_composite);
+    }
+
+    void set_composite_body(
+        int body_index,
```

Comment:
some of these args can be passed in as const

Reply:
I chnaged the int and double arguments to const.